js/views/modals/purchase/Complete.js

```diff
@@ -0,0 +1,126 @@
+import app from '../../../app';
+import loadTemplate from '../../../utils/loadTemplate';
+import BaseVw from '../../baseVw';
+import ChatMessage from '../../../models/chat/ChatMessage';
+import { getEmojiByName } from '../../../data/emojis';
+
+
+export default class extends BaseVw {
+  constructor(options = {}) {
+    super(options);
+    this.options = options;
+
+    this.processingTime = this.options.listing.get('item').processingTime ||
```

Comment:
Could we check that a listing model is passed in as an option. It makes the dependency clearer, since I don't think it would gracefully handle the scenario of the listing not being passed in.